Dockerfile

```diff
@@ -0,0 +1,32 @@
+# To build one auto-instrumentation image for dotnet, please:
+#  - Download your dotnet auto-instrumentation artefacts to `/autoinstrumentation` directory. This is required as when instrumenting the pod,
+#    one init container will be created to copy the files to your app's container.
+#  - Grant the necessary access to the files in the `/autoinstrumentation` directory.
+#  - Following environment variables are injected to the application container to enable the auto-instrumentation.
+#    CORECLR_ENABLE_PROFILING=1
+#    CORECLR_PROFILER={918728DD-259F-4A6A-AC2B-B85E1B658318}
+#    CORECLR_PROFILER_PATH=%InstallationLocation%/linux-x64/OpenTelemetry.AutoInstrumentation.Native.so # for glibc based images
```

Comment:
Even with these just being placeholders I think it would be clear if the `$` syntax is used instead. of `%` since these are Linux images.

Reply:
1. It is just comments and it is understandable.
2. If you really think that is worth to do, we should open PR in the upstream repository first.

Reply:
Not worth the trouble: it is a nit anyway.